package.json

```diff
@@ -27,20 +27,20 @@
     "@guardian/eslint-config-typescript": "^0.4.2",
     "@guardian/prettier": "^0.4.2",
     "@types/aws-lambda": "^8.10.119",
-    "@types/jest": "^26.0.20",
+    "@types/jest": "^29.5.12",
```

Comment:
is there no types/jest 29.7.0?

Reply:
no - https://www.npmjs.com/package/@types/jest

Reply:
do you want to update it in DefinitelyTyped? 😀  there may be no type changes